### DISCUSSION TITLE
7m.pl costs 3$ now?

### DISCUSSION THREAD

**Discussion by DuckyGit** [Cvolton/GMDprivateServer#735] [General]
So i try to register on 7m.pl and it says i need to pay $3.

Is there any other 7m.pl alternatives i can use? (except 5v.pl since it redirects to 7m.pl and freewha since its broken)

**Comment by DarkGD123** [comment]
GDPS Free Hosting By MathieuAR

**Reply by klimer49** [reply]
by stupid idiot and animal

**Reply by GDenisC** [reply]
hi klimer

**Reply by matt7ress** [reply]
hi denisc

**Reply by FedotovDev8801** [reply]
> by stupid idiot and animal

whoa, chill out

**Comment by DarkGD123** [comment]
And 000webhost.com

**Reply by notqoid** [reply]
doesn't work.

**Reply by crypton1871** [reply]
It works, already hosted gdps on it

**Reply by system32149** [reply]
It shut down a while ago.

**Comment by DuckyGit** [comment]
@DarkGD123 Im not gonna host on some VPS i cant access since i dont have a phone number on discord and 000webhost doesnt work.

**Comment by DuckyGit** [comment]
000webhost has a database version GDPS desnt support

**Comment by DarkGD123** [comment]
> @DarkGD123 Im not gonna host on some VPS i cant access since i dont have a phone number on discord and 000webhost doesnt work.

Well go waste money on paid hosting then, no offense

**Comment by ocucor** [comment]
you can use freewha but you might wanna make sure it stays active (has traffic from and to the main website, not the database)

**Reply by notqoid** [reply]
freewha only works for registered domain names

**Comment by DarkGD123** [comment]
well freewha is good but its ftp system is bad but still useful

**Comment by ocucor** [comment]
better than nothing yk

**Comment by DarkGD123** [comment]
just use freewha its pretty nice (just make it alive though)

**Comment by DuckyGit** [comment]
sorry but i now cant access the freewha ftp system

**Comment by DarkGD123** [comment]
> sorry but i now cant access the freewha ftp system

Yeah their ftp system is bad

**Reply by anonymousGDplayer** [reply]
ehhh

**Comment by Jecket22** [comment]
Can't really recommend any free hosts, most I've tried would block geometry dash from accessing the server. So best I would just use a paid webhost, even $.5 hosts are good enough for a gdps. I personally use a virtual private server to have more control over everything, which is also just around $3/month.

**Comment by Cvolton** [comment]
I've enabled the Discussions feature now so if you wish to continue this, I recommend you move over there.
I'm closing this issue as it isn't really relevant to the core project

**Comment by RXNationGMG** [comment]
> GDPS Free Hosting By MathieuAR

Welp GDPS Free Hosting Is Dead

**Comment by ocucor** [comment]
...dead for now. mathieu is trying to get it up again.

**Comment by Jecket22** [comment]
@Cvolton mind you closing this convo

**Comment by ShreyPall** [comment]
https://discord.gg/xEct4reZzx

Join that discord and go to the create gdps channel and type in !c

only works for levels 5 people until mathie gets the servers back up

**Reply by ShreyPall** [reply]
oh @DuckyGit i just saw u cant access server cuz no phone num

so that means u cant access server.
try using ur parents phone number

**Reply by notqoid** [reply]
parents phone number are you insane?

**Comment by notqoid** [comment]
> So i try to register on 7m.pl and it says i need to pay $3.
> 
> Is there any other 7m.pl alternatives i can use? (except 5v.pl since it redirects to 7m.pl and freewha since its broken)

5v.pl. Use a VPN in Poland to bypass the redirect.
5v.pl is basically 7m.pl, but in Polish. You can translate the site to English if you don't understand Polish

also freewha only works for registered domain names

**Reply by cadregusseatus** [reply]
**UPDATED**

Can confirm the freewha part 

UPDATE: After some stupid experimenting, I found a way to get around the 5v/7m paywall

1. Make sure you have a vpn and set it to somewhere in Poland
2. Register at https://5v.pl/
3. Check your inbox for the conformation email, but **totally ignore it**. The link will not work.
4. Reset your password. The activation redirect was removed probably so you couldn't get around the paywall, but resetting your password activates your account anyways.
5. Sign in and register a page and domain. Boom

Don't ask how I discovered this, thanks

**Comment by ocucor** [comment]
I would suggest using [Google Cloud Platform](https://cloud.google.com/free)'s free tier, it requires a credit card to sign up but it doesn't charge you until you enable billing once your $300 credits have been used up.

I use it to host my gdps along with many other projects so i would suggest checking it out :P

**Comment by RXNationGMG** [comment]
I'm using [CrystalCloud](https://crystalcloud.xyz) It Works Fine For Me :)

**Reply by arandomginger09** [reply]
why does it need to join servers for me on discord.....

**Reply by SyeGD** [reply]
I'm gonna try that out.

**Reply by FERFAIL-TXZ** [reply]
was

**Comment by xpureiosx** [comment]
I was pretty annoyed too so I looked around for another PHP webhosting and found infinityfree.com. Thank me later.

**Reply by MegaSa1nt** [reply]
infinityfree doesn't like game services (like gdps) and can block your account

**Comment by ocucor** [comment]
Dog this is an old ahh post

Also infinityfree doesn’t work as intended (as stated earlier iirc?)

**Comment by PMDES** [comment]
I'm a tad bit late, but I would suggest QuartzCloud. It's not very large, so the servers will be much faster than Mathieu's GDPSFH. QuartzCloud also has 5 GB of space while GDPSFH only has 3. If you just want a small GDPS to test stuff on, you could also consider DimisAIO, but it's a bit more complicated than GDPSFH and QuartzCloud.

**Comment by FERFAIL-TXZ** [comment]
Does anyone knows something about 7m.pl? Now the page is down, 6 days already

**Comment by flessan** [comment]
Welp, Gdpsfh will be shutdown maybe..
This may require paying for hosting.

**Comment by kdev-official** [comment]
You Can Host Here : https://infinityfree.com

**Reply by MegaSa1nt** [reply]
this hosting is actively banning gdpss, because gdpss violate their ToS
